### PR TITLE
Pynode datasheet

### PR DESCRIPTION
**Pull request by gijsio**

### CODE REVIEW

**Comment by gijsio** [comment]
The pages are not completely done yet, but lets get the datasheets out there

File: content/datasheets/accessories/pynodecase.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: "Pynode Case"
+---
+
+The Pynode case is designed to enclose your Pynode sensors and allows them to be
```
Comment (peter-pycom) [inline content/datasheets/accessories/pynodecase.md]: To be what?

File: content/datasheets/pynode/_index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+title: "Sensors"
+---
+
+This section contains all of the datasheets for the Pynode sensors
```
Comment (peter-pycom) [inline content/datasheets/pynode/_index.md]: Pynode+

File: content/datasheets/pynode/pynodeair.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+The datasheet of the Pynode+ Air is available as a [PDF File](/gitbook/assets/specsheets/datasheet_pynode_air.pdf)
+
+
+### Dimensions
+
+![](/gitbook/assets/pynodeair.png)
+
+## Pinout
+
+the Pynode+ Air functionality is only accessible over Bluetooth.
```
Comment (peter-pycom) [inline content/datasheets/pynode/pynodeair.md]: All things being equal, I  prefer positive language. This isnot a limitation,  but a deliberate design decision: remove "only"

File: content/datasheets/pynode/pynodeair.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+### Dimensions
+
+![](/gitbook/assets/pynodeair.png)
+
+## Pinout
+
+the Pynode+ Air functionality is only accessible over Bluetooth. 
+
+
+> Be gentle when plugging and unplugging from the USB connector. Whilst the USB connector is soldered and is relatively strong, if it breaks off it can be very difficult to fix.
```
Comment (peter-pycom) [inline content/datasheets/pynode/pynodeair.md]: USB connector?!

File: content/datasheets/pynode/pynodeairq.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+The datasheet of the Pynode+ Air is available as a [PDF File](/assets/gitbook/specsheets/datasheet_pynode_airQ.pdf)
+
+### Dimensions
+
+![](/gitbook/assets/pynodeair.png)
+
+## Pinout
+
+The Pynode+ Air Quality functionality is only accessible over Bluetooth.
```
Comment (peter-pycom) [inline content/datasheets/pynode/pynodeairq.md]: -only

File: content/datasheets/pynode/pynodeairq.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+### Dimensions
+
+![](/gitbook/assets/pynodeair.png)
+
+## Pinout
+
+The Pynode+ Air Quality functionality is only accessible over Bluetooth. 
+
+> Be gentle when plugging and unplugging from the USB connector. Whilst the USB connector is soldered and is relatively strong, if it breaks off it can be very difficult to fix.
```
Comment (peter-pycom) [inline content/datasheets/pynode/pynodeairq.md]: ?

**Comment by peter-pycom** [comment]
Amplify build is not accessible

**Review by peter-pycom — DISMISSED** [review]
See comments

File: content/datasheets/pynode/pynodeair.md (hunk 3 above)
Comment (peter-pycom) [reply content/datasheets/pynode/pynodeair.md]: still

Comment (peter-pycom) [reply content/datasheets/pynode/pynodeair.md]: also the -> The

**Review by peter-pycom — APPROVED** [review]
still 'only', otherwise approved

**Pull request merged**